CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+
+* BREAKING: Content Security Policy forbids unsafe-inline script-src and data: image-src. It provides a nonce generator. Apps that can't support this will need to amend their CSP configuration in an initializer.
```

Comment:
Is it worth linking to a doc explaining more about this initializer?

Reply:
Yup, that's a good shout. I thought in practice that no apps would need to do anything since I'd sorted them out the frontend apps. But now I've discovered the leak into the admin apps that seems a good idea.

Reply:
I'm going to get an example together before updating this item.

Reply:
I've applied this now.